paths.go

```diff
@@ -44,12 +44,11 @@ func newPaths(os, arch string, o Options) (p *Paths, err error) {
 	p.initAstilectronDirectory()
 	p.astilectronApplication = filepath.Join(p.astilectronDirectory, "main.js")
 	p.astilectronDownloadSrc = AstilectronDownloadSrc()
-	p.astilectronDownloadDst = filepath.Join(p.vendorDirectory, fmt.Sprintf("astilectron-v%s.zip", VersionAstilectron))
+	p.astilectronDownloadDst = filepath.Join(p.vendorDirectory, fmt.Sprintf("astilectron-%s-%s-v%s.zip", os, arch, VersionAstilectron))
 	p.astilectronUnzipSrc = filepath.Join(p.astilectronDownloadDst, fmt.Sprintf("astilectron-%s", VersionAstilectron))
 	p.electronDirectory = filepath.Join(p.vendorDirectory, "electron")
```

Comment:
It's working correctly because by chance filenames between electron's os versions doesn't collide. But that may change in the future. Imagine that the Windows version of electron have a `filename.json` and the MacOS version too. If you install the Windows version first, then the MacOS one, the MacOS one will overwrite the Windows one. And it won't work in Windows anymore.

I think it's more secure to also split the electron's directory by os/arch.